### PR TITLE
Add ceremony models

### PR DESCRIPTION
**Pull request by Palesa2lo**
Adding Ceremony Models , there are no pictures really.

### CODE REVIEW

File: src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+            {
+                localVarRequestOptions.HeaderParameters.Add("Accept", localVarAccept);
+            }
+
+            localVarRequestOptions.Data = certifyingCeremonyRequest;
+
+            localVarRequestOptions.Operation = "CertifyingCeremonyApi.CertifyingCeremony";
+            localVarRequestOptions.OperationIndex = operationIndex;
+
+
```
Comment (PetriJordaan) [inline src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs:261]: RA3 - Spaaaaace

File: src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+        /// <returns>Task of ApiResponse (CertifyingCeremonyResponse)</returns>
+        public async System.Threading.Tasks.Task<SigniFlow.CertifyCeremonyAPI.Client.ApiResponse<CertifyingCeremonyResponse>> CertifyingCeremonyWithHttpInfoAsync(CertifyingCeremonyRequest certifyingCeremonyRequest, int operationIndex = 0, System.Threading.CancellationToken cancellationToken = default(global::System.Threading.CancellationToken))
+        {
+            // verify the required parameter 'certifyingCeremonyRequest' is set
+            if (certifyingCeremonyRequest == null)
+            {
+                throw new SigniFlow.CertifyCeremonyAPI.Client.ApiException(400, "Missing required parameter 'certifyingCeremonyRequest' when calling CertifyingCeremonyApi->CertifyingCeremony");
+            }
+
+
```
Comment (PetriJordaan) [inline src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs:306]: RA3 - Spaaaaace

File: src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+            {
+                localVarRequestOptions.HeaderParameters.Add("Accept", localVarAccept);
+            }
+
+            localVarRequestOptions.Data = certifyingCeremonyRequest;
+
+            localVarRequestOptions.Operation = "CertifyingCeremonyApi.CertifyingCeremony";
+            localVarRequestOptions.OperationIndex = operationIndex;
+
+
```
Comment (PetriJordaan) [inline src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs:336]: RA3 - Spaaaaace

File: src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+                Exception _exception = this.ExceptionFactory("CertifyingCeremony", localVarResponse);
+                if (_exception != null)
+                {
+                    throw _exception;
+                }
+            }
+
+            return localVarResponse;
+        }
+
```
Comment (PetriJordaan) [inline src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs:351]: RA3 - Spaaaaace

File: src/SigniFlow.Connect/Model/CertifyingCeremonyRequest.cs [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+        /// To validate all properties of the instance
+        /// </summary>
+        /// <param name="validationContext">Validation context</param>
+        /// <returns>Validation Result</returns>
+        IEnumerable<ValidationResult> IValidatableObject.Validate(ValidationContext validationContext)
+        {
+            yield break;
+        }
+    }
+
```
Comment (PetriJordaan) [inline src/SigniFlow.Connect/Model/CertifyingCeremonyRequest.cs:228]: RA3 - Spaaaaace

File: src/SigniFlow.Connect/Model/CertifyingCeremonyResponse.cs [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+        /// To validate all properties of the instance
+        /// </summary>
+        /// <param name="validationContext">Validation context</param>
+        /// <returns>Validation Result</returns>
+        IEnumerable<ValidationResult> IValidatableObject.Validate(ValidationContext validationContext)
+        {
+            yield break;
+        }
+    }
+
```
Comment (PetriJordaan) [inline src/SigniFlow.Connect/Model/CertifyingCeremonyResponse.cs:103]: RA3 - Spaaaaace

File: src/SigniFlow.Connect.Test/Api/CertifyingCeremonyApiTests.cs [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+            //Assert.IsType<CertifyingCeremonyApi>(instance);
+        }
+
+        /// <summary>
+        /// Test CertifyingCeremony
+        /// </summary>
+        [Fact]
+        public void CertifyingCeremonyTest()
+        {
+            // TODO uncomment below to test the method and replace null with proper value
```
Comment (Riccorbypro) [inline src/SigniFlow.Connect.Test/Api/CertifyingCeremonyApiTests.cs:64]: Please don't leave unit tests with commented out code.

File: src/SigniFlow.Connect.Test/Model/CertifyingCeremonyRequestTests.cs [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+    /// Please update the test case below to test the model.
+    /// </remarks>
+    public class CertifyingCeremonyRequestTests : IDisposable
+    {
+        // TODO uncomment below to declare an instance variable for CertifyingCeremonyRequest
+        //private CertifyingCeremonyRequest instance;
+
+        public CertifyingCeremonyRequestTests()
+        {
+            // TODO uncomment below to create an instance of CertifyingCeremonyRequest
```
Comment (Riccorbypro) [inline src/SigniFlow.Connect.Test/Model/CertifyingCeremonyRequestTests.cs:39]: Please don't leave unit tests with commented out code.

File: src/SigniFlow.Connect.Test/Model/CertifyingCeremonyRequestTests.cs [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+            // Cleanup when everything is done.
+        }
+
+        /// <summary>
+        /// Test an instance of CertifyingCeremonyRequest
+        /// </summary>
+        [Fact]
+        public void CertifyingCeremonyRequestInstanceTest()
+        {
+            // TODO uncomment below to test "IsType" CertifyingCeremonyRequest
```
Comment (Riccorbypro) [inline src/SigniFlow.Connect.Test/Model/CertifyingCeremonyRequestTests.cs:54]: Please don't leave unit tests with commented out code.

File: src/SigniFlow.Connect.Test/Model/CertifyingCeremonyResponseTests.cs [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+        }
+
+        /// <summary>
+        /// Test an instance of CertifyingCeremonyResponse
+        /// </summary>
+        [Fact]
+        public void CertifyingCeremonyResponseInstanceTest()
+        {
+            // TODO uncomment below to test "IsType" CertifyingCeremonyResponse
+            //Assert.IsType<CertifyingCeremonyResponse>(instance);
```
Comment (Riccorbypro) [inline src/SigniFlow.Connect.Test/Model/CertifyingCeremonyResponseTests.cs]: Please don't leave unit tests with commented out code.

**Review by Signi-TK — APPROVED** [review]

**Review by Riccorbypro — CHANGES_REQUESTED** [review]
Please implement the missing unit tests

File: src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs (hunk 1 above)
Comment (Riccorbypro) [reply src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs:261]: This is in generated code

File: src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs (hunk 2 above)
Comment (Riccorbypro) [reply src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs:306]: This is in generated code

File: src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs (hunk 3 above)
Comment (Riccorbypro) [reply src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs:336]: This is in generated code

File: src/SigniFlow.Connect/Model/CertifyingCeremonyRequest.cs (hunk 5 above)
Comment (Riccorbypro) [reply src/SigniFlow.Connect/Model/CertifyingCeremonyRequest.cs:228]: This is in generated code

File: src/SigniFlow.Connect/Model/CertifyingCeremonyResponse.cs (hunk 6 above)
Comment (Riccorbypro) [reply src/SigniFlow.Connect/Model/CertifyingCeremonyResponse.cs:103]: This is in generated code

File: src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs (hunk 4 above)
Comment (Riccorbypro) [reply src/SigniFlow.Connect/Api/CertifyingCeremonyApi.cs:351]: This is in generated code

**Review by marko3645 — APPROVED** [review]

**Review by PetriJordaan — APPROVED** [review]

**Review by nataleap — APPROVED** [review]

**Review by MorneWilken — APPROVED** [review]

**Pull request merged**